Bindings/Python/OpenSenseExample/OpenSense_CalibrateModel.py

```diff
@@ -22,14 +22,14 @@
 
 # Import OpenSim libraries
 import opensim as osim
-
+import math.pi as pi
```

Comment:
This line doesn't work for me (`No module named 'math.pi'; 'math' is not a package`). Consider `from math import pi` instead, and then `pi` will be defined too